### PR TITLE
Make the div an anchor

### PR DESCRIPTION
**Pull request by LukeHendrick**
Downloaded the engine keyboard only, I was unable to close the modal. Swapped the div out for a button and made it transparent to be a little more accessible

### CODE REVIEW

**Comment by YuriSizov** [comment]
Thanks for opening a PR. I think it should be an anchor then, to match the rest of our buttons and avoid button-specific behavior and styling from browsers.

**Comment by LukeHendrick** [comment]
Swapped for an anchor!

File: _layouts/download-3.html [hunk 1]
```diff
@@ -330,6 +330,17 @@ <h2>Preview builds</h2>
...
+			// Support button enter and space interactions
+			switch (event.code.toUpperCase()) {
+				case "ENTER":
+				case "SPACE":
+					thankYouWrapper.style.display = 'none';
+					break;
+				default:
+					break;
+			}
+		});
```
Comment (YuriSizov) [inline _layouts/download-3.html:343]: Why do we need this? Pressing enter already activates the link. Pressing space scrolls the page, but that's normal for web.

**Comment by YuriSizov** [comment]
Sorry for the delay! The layout change seems good to me, but I'm not sure about the added code.

**Comment by coppolaemilio** [comment]
Superseded by https://github.com/godotengine/godot-website/commit/2167a80780b73ffdcd1910faa83405a1b3e7defb.
Thanks for the PR!

**Pull request closed without merging**